lib/contentTypeParser.js

```diff
@@ -163,12 +165,15 @@ ContentTypeParser.prototype.run = function (contentType, handler, request, reply
   }
 
   function done (error, body) {
-    if (error) {
-      reply.send(error)
-    } else {
-      request.body = body
-      handler(request, reply)
-    }
+    // We cannot use resource.bind() because it is broken in node v12.
+    resource.runInAsyncScope(() => {
```

Comment:
does this introduce noticeable overhead for people who do not use ALS at all?

Reply:
I've done a few tests in the past and it was negligible. The major cost is the allocation of an additional object that we throw away after it's all said and done.